definitions/ext-dell_avamar/dashboard.json

```diff
@@ -0,0 +1,233 @@
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 1,
+            "height": 2
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": " SELECT count(`clientName`) as 'Failed' FROM ( SELECT uniqueCount(clientName)  FROM avamar WHERE key = 'activity' WHERE type = 'Scheduled Backup' FACET clientName,datadomain limit max WHERE stateID = 2)"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Backup exceeding 24 hours",
+          "layout": {
+            "column": 2,
+            "row": 1,
+            "width": 1,
+            "height": 2
+          },
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": " SELECT uniqueCount(`clientName`) as 'Clients' FROM (SELECT filter(latest(elapsedTime)/60/60/24, where elapsedTime/60/60/24 > 1) as 'Over 24 hours',latest(elapsedTime)/60/60 as 'run time Hours',latest(clientName),latest(overtimeOption),latest(state),latest(stateID),latest(description),latest(errorCode),latest(codeReason),latest(stats.clientOs),latest(stats.clientVersion),latest(timeZone),latest(type),latest(completedDate),latest(endStartTime),latest(`endTime`),latest(dispensedTime),latest(datadomain),latest(proxyName),latest(retention),latest(scheduleName),latest(allowOvertime),latest(datasetFqdn),latest(groupFqdn) FROM avamar WHERE key = 'activity' FACET clientName limit max where overtimeOption = 'NEVER')"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Running backups",
+          "layout": {
+            "column": 3,
+            "row": 1,
+            "width": 3,
+            "height": 2
+          },
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT uniqueCount(clientName)  FROM avamar WHERE key = 'activity' where type = 'Scheduled Backup' FACET clientName,state limit max WHERE stateID = 3  TIMESERIES 2 minutes "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
```

Comment:
I need to understand the difference between true and false, I'm just exporting the dashboard and not sure what it does. However I'm fine with both options

Reply:
Usually, we always use `ignoreTimeRange: false` as the default value.
It means that a time clause is automatically added to all queries with the global time range you have in New Relic. This is the better option as it allows users to choose the time range and use several.

But there are cases when you want a fixed amount of time and ignore the one that the user has selected; only in these cases, you have to define `ignoreTimeRange: true`.
In that case that you have a title of `Running backups for 24h` and a `SINCE 24 hours ago` it must be set to true, for other panels false it's ok.